docs/interfaces.md

```diff
@@ -385,3 +391,9 @@ Can manage timeservers directly separate from ``config core``.
 Can access the tpm device /dev/tpm0.
 
 * Auto-Connect: no
+
+### udisks2
+
+Can detect, mount, and unmount USB storage devices and SD cards.
```

Comment:
udisks2 can do much more than this though, right? Also, this should be phrased from the perspective of the person plugging, not the person slotting. I suggest: `Can access snaps providing the udisks2 interface which gives privileged access to storage on the device`